packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -58,28 +64,166 @@ class AllocationTableViewState extends State<AllocationTableView>
+    final fixFontStyle = fixedFontStyle(context);
+
+    final allocationSample = controller.allocationSamples.singleWhere(
+      (element) => element.classRef.id == classRef.id,
+      orElse: () => null,
+    );
+
+    // Nothing tracked.
+    if (allocationSample == null) {
+      return [
+        Padding(
+          padding: const EdgeInsets.only(bottom: 5),
+          child: Text('empty', style: fixFontStyle),
+        )
+      ];
+    }
+
+    final smallerFixedFont = smallerFixedFontStyle(context);
+
+    final subTitles = <ExpansionTile>[];
+
+    for (var stacktrace in allocationSample.stacktraces) {
+      final callStackEntries = <Widget>[];
+
+      callStackEntries.add(Text(
+        '${stacktrace.stacktraceDisplay(maxLines: stacktrace.showFullStacktrace ? -1 : 4)}',
+        style: smallerFixedFont,
+      ));
+      if (stacktrace.stackDepth > 4 && !stacktrace.showFullStacktrace) {
+        callStackEntries.add(
+          InkWell(
+            onTap: () {
+              stacktrace.showFullStacktrace = true;
+              // TODO(terry): Use a value notifier.
+              setState(() {});
+            },
+            child: Padding(
+              padding: const EdgeInsets.only(bottom: 10),
+              child: Text('More ...', style: smallerFixedFont),
+            ),
+          ),
+        );
+      }
+
+      final stackTraceWidget = Container(
+        padding: const EdgeInsets.symmetric(horizontal: 80),
+        alignment: Alignment.topLeft,
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: callStackEntries,
+        ),
+      );
+
+      subTitles.add(
+        ExpansionTile(
+          tilePadding: const EdgeInsets.only(left: 80, right: 20),
+          childrenPadding: EdgeInsets.zero,
+          title: Text(
+            '${allocationSample.classRef.name} '
+            '@${prettyTimestamp(stacktrace.timestamp)}',
+            style: fixFontStyle,
+          ),
+          children: [stackTraceWidget],
+        ),
+      );
+    }
+
+    return subTitles;
+  }
+
+  ExpansionTile _createExpansionTile(int index) {
+    final fixFontStyle = fixedFontStyle(context);
+
+    if (controller.trackAllocations.isNotEmpty) {
+      // Show what is to be tracked.
+      final trackingClasses = controller.trackAllocations.keys.toList();
+      final className = trackingClasses[index];
+      final classRef = controller.trackAllocations[className];
+      final childWidgets = allTrackedCreations(classRef);
+      return ExpansionTile(
+        leading: const Icon(Icons.track_changes),
+        tilePadding: const EdgeInsets.symmetric(horizontal: 20),
+        childrenPadding: EdgeInsets.zero,
+        title: Text(className, style: fixFontStyle),
+        children: childWidgets,
+      );
+    } else {
+      return null;
+    }
+  }
+
+  Widget trackedClassesList() {
+    return controller.trackAllocations.isEmpty
+        ? const Padding(
+            padding: EdgeInsets.only(top: 15),
```

Comment:
use the existing size constants. there is a normalSpacing that is 16 and small that is 8. Using those will keep the UI a little more consistent.

Reply:
Re-wrote using TreeTable not needed.